api/v1beta1/disruption_webhook_test.go

```diff
@@ -514,7 +514,6 @@ var _ = Describe("Disruption", func() {
 
 			JustBeforeEach(func() {
 				newDisruption = makeValidDiskFailureDisruption()
-				controllerutil.AddFinalizer(newDisruption, chaostypes.DisruptionFinalizer)
```

Comment:
Why we remove the finalizer here?

Reply:
It wasn't relevant to the testing at all, seems to have only been included because this was copy-pasted from the finalizer tests at the top of the file